mne/preprocessing/ica.py

```diff
@@ -1133,9 +1165,13 @@ def find_bads_ecg(self, inst, ch_name=None, threshold=None, start=None,
             The name of the channel to use for ECG peak detection.
             The argument is mandatory if the dataset contains no ECG
             channels.
-        threshold : float
+        threshold : float | str | None
```

Comment:
I think we should deprecate the None option and move to 'auto' in next versions. In 0.21 we get a warning to set it to auto to avoid warning and in 0.22 we make 'auto' the default and deprecate None.

Reply:
ok